README.md

```diff
@@ -34,19 +34,20 @@ The easiest way to manage multiple node versions in development is to use
 ## Get started
 
 * type `yarn` to install all dependencies
-* type `yarn dev:amo` to start a dev server
+* type `yarn dev:amo:dev` to start a local dev server that connects to a
```

Comment:
Oh, that's a good call, makes it easier for contributors to get started without Docker. 👍